hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestLeaseRecovery.java

```diff
@@ -265,7 +265,7 @@ public void testRecoveryWithoutHsyncHflushOnLastBlock() throws Exception {
   @Test
   public void testOBSRecoveryShouldFail() throws Exception {
     OzoneBucket obsBucket = TestDataUtil.createVolumeAndBucket(client,
-        "vol2", "obs", BucketLayout.OBJECT_STORE);
+        "vol2", "obs", BucketLayout.OBJECT_STORE, null);
```

Comment:
you can keep the current createVolumeAndBucket and add a new one, so these unrelevant unit test files can not be touched.

Reply:
Noted, made the required changes, thank you!